__tests__/app.test.js

```diff
@@ -517,3 +517,51 @@ describe("/api/reviews/:review_id/comments", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    test("returns a status code of 204 and no body content", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          expect(body).toEqual({});
+        });
+    });
+    test("removes correct comment from the database", () => {
+      //This test works only with this test data. As we don't have a /api/comments API to make a get request to, this test manually checks that the review was deleted through its association with review_id 2
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .then(({ body }) => {
+          const lengthBefore = body.comments.length;
+          return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+            .then(() => {
+              return request(app)
+                .get("/api/reviews/2/comments")
+                .then(({ body }) => {
+                  const lengthAfter = body.comments.length;
+                  expect(lengthAfter).toBe(lengthBefore - 1);
+                });
```

Comment:
this is a nice test but it's not really necessary as test one passes but nice thought!